backend/api/Database/Models/InspectionFindings.cs

```diff
@@ -1,22 +1,46 @@
-﻿using Microsoft.EntityFrameworkCore;
-
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Api.Controllers.Models;
 #pragma warning disable CS8618
 namespace Api.Database.Models
 {
-    [Owned]
     public class InspectionFindings
     {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public string Id { get; set; }
 
         public string RobotName { get; set; }
 
         public string InspectionDate { get; set; }
 
         public string Area { get; set; }
 
-        public string InspectionId { get; set; }
-
-        public string FindingsTag { get; set; }
-
+        public string IsarStepId { get; set; }
+
+        public string Findings { get; set; }
+
+        public string MissionRunId { get; set; }
```

Comment:
If we have a link from Inspection, I dont think we need MissionRunId

Reply:
Removed, it was added earlier when we have owned entities. We dont need it anymore.